files/jsondata/L10n-SVG.json

```
@@ -360,15 +379,17 @@
     "ja": "プレゼンテーション属性",
     "pt-BR": "Atributos de apresentação",
     "ru": "Атрибуты показа",
-    "zh-CN": "外观属性"
+    "zh-CN": "外观属性",
+    "zh-TW": "外觀屬性"
```

or using `表现`: 

- https://zh.wikipedia.org/zh-tw/%E8%A1%A8%E7%8E%B0%E5%B1%82%E7%8A%B6%E6%80%81%E8%BD%AC%E6%8D%A2
- https://developer.mozilla.org/zh-CN/docs/Web/SVG/Attribute/Presentation


```suggestion
    "zh-CN": "表现属性",
    "zh-TW": "表現屬性"
```